Subject: Pin snowball to the commit that was main on the typesense release day
Origin: DHI-internal
Description: snowballstem/snowball renamed the default branch from master to
 main, so Bazel 6's branch = "master" fetch fails. Pin the commit that was
 main on the v29.1 and v30.2 release day (2026-04-19).
Commit: https://github.com/snowballstem/snowball/commit/a288f069d325bc5a64b280c01069a632bed58205

--- a/WORKSPACE
+++ b/WORKSPACE
@@ -341,7 +341,7 @@
 new_git_repository(
     name= "snowball",
     build_file = "//bazel:snowball.BUILD",
-    branch = "master",
+    commit = "a288f069d325bc5a64b280c01069a632bed58205",
     remote = "https://github.com/snowballstem/snowball.git"
 )
 
